ArduCopter/RC_Channel.h

```diff
@@ -42,6 +42,9 @@ class RC_Channels_Copter : public RC_Channels
         return &obj_channels[chan];
     }
 
+    // allow disabling AP_Arming throttle check
```

Comment:
not a big deal but this should probably be, "returns true if throttle arming checks should be run".. not a blocker though!

Reply:
fixed